src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -23,6 +19,7 @@
  */
 public class RepositoryApiClientImpl implements RepositoryApiClient, AutoCloseable {
     private Map<String, String> defaultHeaders;
+    private Map<String, String> gzipCompressionHeaders = new HashMap<>();
```

Comment:
this doesn't need to be a class variable, could create it in the constructor instead.

Reply:
ok done